x-pack/platform/plugins/shared/ml/common/types/capabilities.ts

```diff
@@ -112,7 +112,7 @@ export function getDefaultCapabilities(): MlCapabilities {
 
 const alertingFeatures = Object.values(ML_ALERT_TYPES).map((ruleTypeId) => ({
   ruleTypeId,
-  consumers: [PLUGIN_ID, ALERTING_FEATURE_ID],
+  consumers: [PLUGIN_ID, ALERTING_FEATURE_ID, ...DEPRECATED_ALERTING_CONSUMERS],
```

Comment:
This specifies that the `observability` consumer is a valid consumer for this alerting feature when this privilege is granted. The privilege in this scenario is `ML`

I named this `DEPRECATED_ALERTING_CONSUMERS` because I wanted a name that was more semantic that included a bit of information about why the consumer was added here. I thought that would be more easily understood than if I included a random `OBSERVABILITY_FEATURE_ID` here.